core/chains/terra/terratxm/key_wrapper.go

```diff
@@ -0,0 +1,56 @@
+package terratxm
```

Comment:
This implementation used the `TransmissionSigner` interface and was located with the rest of the Terra specific codebase at: https://github.com/smartcontractkit/chainlink-terra

It's now moved back to the CL core repo, the interface was removed which is a step back IMO.